feat: added basic example

rajatagarwal-ibm opened the pull request:
### Description

Basic example to create COS instance and bucket

### Types of changes in this PR

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [ ] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [x] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version. 

Your notes helps the merger write the commit message for the PR that is published in the release notes for the module.

---

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

shemau reviewed (changes requested):
This is essentially a copy of the one-rate example, just without the COS plan and storage class, which means maintaining duplicate code moving forward.
I wonder if there is a better balance, maybe moving one-rate from an example to documentation. The cost of testing every possible combination of every COS provider input is prohibitive.

rajatagarwal-ibm commented:
@shemau addressed your comments. I added `access_tags` because I reused the `setupOptions` which needed that value. 

Also, right now most of the examples and tests are refactored to minimal resource consumption, epic in progress https://github.ibm.com/GoldenEye/issues/issues/4615.

shemau reviewed (approved):
The requested changes look good.

terraform-ibm-modules-ops commented:
:tada: This PR is included in version 6.6.1 :tada:

The release is available on [GitHub release](https://github.com/terraform-ibm-modules/terraform-ibm-cos/releases/tag/v6.6.1)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
